draft-barnes-acme.md

```diff
@@ -1646,13 +1646,16 @@ The client serializes the validation object to UTF-8, then uses its account
 private key to sign a JWS with the serialized JSON object as its payload.  This
 JWS is NOT REQUIRED to have the "nonce" header parameter.
 
-The client will generate a self-signed certificate with the subject's
-organizationName field set to the "signature" value from the JWS, i.e., the
-base64-encoded signature value, and a subjectAlternativeName extension
-containing a single dNSName of "\<token\>.acme.invalid".  The client will then
-configure the TLS server at the domain such that when a handshake is initiated
-with the Server Name Indication extension set to "\<token\>.acme.invalid", the
-generated test certificate is presented.
+The client will compute Z, the SHA-256 of the "signature" value from the JWS,
+i.e., the base64-encoded signature value.  Z is encoded in hexidecimal form.
+
+The client will generate a self-signed certificate with the
+subjectAlternativeName extension containing two dNSNames. The first is set to
+"\<token\>.acme.invalid" and the second is set to
+"\<Z[0:32]\>.\<Z[32:64]\>.acme.invalid".  The client will then configure the TLS
+server at the domain such that when a handshake is initiated with the Server
+Name Indication extension set to "\<token\>.acme.invalid", the generated test
+certificate is presented.
```

Comment:
This is duplicative.  Just use the `Z.acme.invalid` name, since the signature covers the token.